frontend/finance-tracker/src/components/ui/calendar.tsx

```
@@ -2,74 +1,524 @@
+    return (
+        <Button
+            className="h-7 w-full truncate text-sm font-medium"
+            variant="ghost"
+            size="sm"
+            onClick={() =>
+                setNavView((prev) => (prev === "days" ? "years" : "days"))
+            }
+        >
+            {navView === "days"
+                ? children
+                : displayYears.from + " - " + displayYears.to}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)